vllm/v1/core/kv_cache_manager.py

```diff
@@ -175,9 +185,6 @@ def allocate_slots(
                 not include the tokens that have already been computed.
             new_computed_blocks: A list of new computed blocks just hitting the
                 prefix caching.
-            num_lookahead_tokens: The number of speculative tokens to allocate.
```

Comment:
Is this removed by accident? @robertgshaw2-redhat

Reply:
No, its updated

Reply:
Woosuk asked to make a new api that does not use `allocate_slots`.

Reply:
I see thanks for your reply @robertgshaw2-redhat